### ISSUE TITLE
[V5] [BUG] When activating Nth selection of Submenu, instead, its Context menu's Nth selection activates

### ISSUE THREAD

**Issue by drkhn1234** [mbnuqw/sidebery#1175]
## Steps to reproduce
- Open Firefox in [new profile](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles)
- Install Sidebery
- Open context menu via shortcut
- Move down to a submenu via "Select up/down" shortcuts
- Enter into the submenu using "Activate selection" shortcut
- Move, by using "Select down" shortcut, "N" times down to a submenu item and activate it using "Activate selection" shortcut.

## Expected behavior
The pertinent submenu item of Nth index activates.

## Actual behavior
Nth item in the Context menu comprising the submenu activates.

## Sidenote
This applies to all context menus, not just the tabs' context menu.

## Debug info

<details>
<summary><b>Addon data</b></summary>
     
```json
{
  "addonVersion": "5.0.0rc4",
  "firefoxVersion": "116.0.2",
  "settings": {
    "nativeScrollbars": true,
    "nativeScrollbarsThin": true,
    "nativeScrollbarsLeft": false,
    "selWinScreenshots": true,
    "updateSidebarTitle": true,
    "markWindow": true,
    "markWindowPreface": "* ",
    "ctxMenuNative": false,
    "ctxMenuRenderInact": false,
    "ctxMenuRenderIcons": false,
    "ctxMenuIgnoreContainers": "",
    "navBarLayout": "vertical",
    "navBarInline": true,
    "navBarSide": "left",
    "hideAddBtn": false,
    "hideSettingsBtn": false,
    "navBtnCount": true,
    "hideEmptyPanels": true,
    "hideDiscardedTabPanels": false,
    "navActTabsPanelLeftClickAction": "scroll",
    "navActBookmarksPanelLeftClickAction": "scroll",
    "navTabsPanelMidClickAction": "discard",
    "navBookmarksPanelMidClickAction": "none",
    "navSwitchPanelsWheel": true,
    "subPanelRecentlyClosedBar": true,
    "subPanelBookmarks": false,
    "subPanelHistory": true,
    "groupLayout": "list",
    "skipEmptyPanels": false,
    "dndTabAct": false,
    "dndTabActDelay": 500,
    "dndTabActMod": "none",
    "dndExp": "hover",
    "dndExpDelay": 500,
    "dndExpMod": "none",
    "dndOutside": "win",
    "dndActTabFromLink": false,
    "dndActSearchTab": true,
    "dndMoveTabs": false,
    "dndMoveBookmarks": false,
    "searchBarMode": "dynamic",
    "searchBookmarksShortcut": "*",
    "searchHistoryShortcut": "!",
    "warnOnMultiTabClose": "none",
    "activateLastTabOnPanelSwitching": true,
    "activateLastTabOnPanelSwitchingLoadedOnly": true,
    "switchPanelAfterSwitchingTab": "always",
    "tabRmBtn": "hover",
    "activateAfterClosing": "next",
    "activateAfterClosingStayInPanel": true,
    "activateAfterClosingGlobal": false,
    "activateAfterClosingNoFolded": false,
    "activateAfterClosingNoDiscarded": true,
    "askNewBookmarkPlace": true,
    "tabsRmUndoNote": false,
    "tabsUnreadMark": true,
    "tabsUpdateMark": "all",
    "tabsUpdateMarkFirst": true,
    "tabsReloadLimit": 5,
    "tabsReloadLimitNotif": true,
    "showNewTabBtns": true,
    "newTabBarPosition": "bottom",
    "tabsPanelSwitchActMove": false,
    "tabsPanelSwitchActMoveAuto": true,
    "tabsUrlInTooltip": "full",
    "newTabCtxReopen": false,
    "moveNewTabPin": "end",
    "moveNewTabParent": "last_child",
    "moveNewTabParentActPanel": true,
    "moveNewTab": "before",
    "moveNewTabActivePin": "start",
    "pinnedTabsPosition": "left",
    "pinnedTabsList": true,
    "pinnedAutoGroup": true,
    "pinnedNoUnload": true,
    "tabsTree": true,
    "groupOnOpen": true,
    "tabsTreeLimit": 3,
    "autoFoldTabs": true,
    "autoFoldTabsExcept": 1,
    "autoExpandTabs": true,
    "autoExpandTabsOnNew": true,
    "rmChildTabs": "folded",
    "tabsChildCount": true,
    "tabsLvlDots": true,
    "discardFolded": true,
    "discardFoldedDelay": 3,
    "discardFoldedDelayUnit": "min",
    "tabsTreeBookmarks": true,
    "treeRmOutdent": "branch",
    "ignoreFoldedParent": true,
    "colorizeTabs": false,
    "colorizeTabsSrc": "domain",
    "colorizeTabsBranches": true,
    "colorizeTabsBranchesSrc": "url",
    "inheritCustomColor": false,
    "hideInact": false,
    "hideFoldedTabs": true,
    "hideFoldedParent": "any",
    "nativeHighlight": true,
    "warnOnMultiBookmarkDelete": "any",
    "autoCloseBookmarks": true,
    "autoRemoveOther": false,
    "highlightOpenBookmarks": true,
    "activateOpenBookmarkTab": true,
    "showBookmarkLen": true,
    "bookmarksRmUndoNote": true,
    "loadBookmarksOnDemand": true,
    "pinOpenedBookmarksFolder": true,
    "oldBookmarksAfterSave": "ask",
    "loadHistoryOnDemand": true,
    "fontSize": "m",
    "animations": true,
    "animationSpeed": "norm",
    "theme": "proton",
    "density": "compact",
    "colorScheme": "ff",
    "sidebarCSS": false,
    "groupCSS": false,
    "snapNotify": true,
    "snapExcludePrivate": false,
    "snapInterval": 0,
    "snapIntervalUnit": "min",
    "snapLimit": 0,
    "snapLimitUnit": "snap",
    "snapAutoExport": false,
    "snapAutoExportType": "json",
    "snapAutoExportPath": "Sidebery/snapshot-%Y.%M.%D-%h.%m.%s",
    "snapMdFullTree": true,
    "hScrollAction": "none",
    "navSwitchPanelsDelay": 122,
    "scrollThroughTabs": "none",
    "scrollThroughVisibleTabs": true,
    "scrollThroughTabsSkipDiscarded": true,
    "scrollThroughTabsExceptOverflow": true,
    "scrollThroughTabsCyclic": true,
    "scrollThroughTabsScrollArea": 0,
    "autoMenuMultiSel": true,
    "multipleMiddleClose": true,
    "longClickDelay": 500,
    "wheelThreshold": false,
    "wheelThresholdX": 10,
    "wheelThresholdY": 60,
    "tabDoubleClick": "none",
    "tabsSecondClickActPrev": true,
    "tabsSecondClickActPrevPanelOnly": false,
    "shiftSelAct": true,
    "activateOnMouseUp": true,
    "tabLongLeftClick": "none",
    "tabLongRightClick": "none",
    "tabCloseMiddleClick": "close",
    "tabsPanelLeftClickAction": "expand",
    "tabsPanelDoubleClickAction": "tab",
    "tabsPanelRightClickAction": "menu",
    "tabsPanelMiddleClickAction": "tab",
    "newTabMiddleClickAction": "new_child",
    "bookmarksLeftClickAction": "open_in_new",
    "bookmarksLeftClickActivate": false,
    "bookmarksLeftClickPos": "default",
    "bookmarksMidClickAction": "open_in_new",
    "bookmarksMidClickActivate": false,
    "bookmarksMidClickRemove": false,
    "bookmarksMidClickPos": "default",
    "syncName": "",
    "syncSaveSettings": true,
    "syncSaveCtxMenu": true,
    "syncSaveStyles": false,
    "syncSaveKeybindings": true,
    "selectActiveTabFirst": false
  },
  "permissions": {
    "allUrls": true,
    "webRequest": true,
    "webRequestBlocking": true,
    "proxy": true,
    "tabHide": true,
    "clipboardWrite": true,
    "history": true,
    "bookmarks": true,
    "downloads": true
  },
  "storage": {
    "size": "232 kb",
    "props": {
      "bookmarksRecentFolders": "31 b",
      "containers": "1.35 kb",
      "contextMenu": "1.54 kb",
      "expandedBookmarkFolders": "76 b",
      "favDomains": "20.3 kb",
      "favHashes": "1.69 kb",
      "favicons": "193 kb",
      "lastSnapTime": "13 b",
      "profileID": "14 b",
      "settings": "4.39 kb",
      "sidebar": "3.78 kb",
      "snapshots": "4.27 kb",
      "tabsDataCache": "944 b",
      "ver": "10 b"
    }
  },
  "sidebar": {
    "nav": [
      "sd-m3CUBD36ZAJV",
      "hdn",
      "rpG2kpSHVfHu",
      "KvQWXJczvH1P",
      "Q6htrSevEsJu",
      "k6whdNiATW8Z",
      "sp-CtcijeKZwpJV",
      "remute_audio_tabs",
      "sd-ILT5Ci70maKV",
      "create_snapshot",
      "settings"
    ],
    "panels": {
      "VghWnYI3IcdZ": {
        "type": 1,
        "id": "VghWnYI3IcdZ",
        "name": "len: 14",
        "iconSVG": "icon_bookmarks",
        "iconIMGSrc": "",
        "iconIMG": "",
        "color": "toolbar",
        "lockedPanel": false,
        "tempMode": false,
        "skipOnSwitching": false,
        "rootId": "menu________",
        "viewMode": "tree",
        "autoConvert": false,
        "srcPanelConfig": null
      },
      "XvDVswFOngKV": {
        "type": 1,
        "id": "XvDVswFOngKV",
        "name": "len: 9",
        "iconSVG": "icon_bookmarks",
        "iconIMGSrc": "",
        "iconIMG": "",
        "color": "toolbar",
        "lockedPanel": false,
        "tempMode": false,
        "skipOnSwitching": false,
        "rootId": "root________",
        "viewMode": "tree",
        "autoConvert": false,
        "srcPanelConfig": null
      },
      "rpG2kpSHVfHu": {
        "type": 2,
        "id": "rpG2kpSHVfHu",
        "name": "len: 7",
        "color": "purple",
        "iconSVG": "icon_tabs",
        "iconIMGSrc": "len: 11",
        "iconIMG": "len: 766",
        "lockedPanel": false,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "none",
        "dropTabCtx": "none",
        "moveRules": [],
        "moveExcludedTo": -1,
        "bookmarksFolderId": "3tq4mRJo1L7z",
        "newTabBtns": [],
        "srcPanelConfig": {
          "id": "h1uM-yvcxCVv",
          "autoConvert": true,
          "tempMode": false,
          "viewMode": "tree"
        }
      },
      "KvQWXJczvH1P": {
        "type": 2,
        "id": "KvQWXJczvH1P",
        "name": "len: 7",
        "color": "red",
        "iconSVG": "icon_tabs",
        "iconIMGSrc": "len: 10",
        "iconIMG": "len: 626",
        "lockedPanel": false,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "none",
        "dropTabCtx": "none",
        "moveRules": [],
        "moveExcludedTo": -1,
        "bookmarksFolderId": -1,
        "newTabBtns": [],
        "srcPanelConfig": null
      },
      "Q6htrSevEsJu": {
        "type": 2,
        "id": "Q6htrSevEsJu",
        "name": "len: 4",
        "color": "turquoise",
        "iconSVG": "icon_tabs",
        "iconIMGSrc": "",
        "iconIMG": "",
        "lockedPanel": false,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "none",
        "dropTabCtx": "none",
        "moveRules": [],
        "moveExcludedTo": -1,
        "bookmarksFolderId": -1,
        "newTabBtns": [],
        "srcPanelConfig": null
      },
      "HW53-JvVMmdZ": {
        "type": 2,
        "id": "HW53-JvVMmdZ",
        "name": "len: 18",
        "color": "toolbar",
        "iconSVG": "icon_tabs",
        "iconIMGSrc": "",
        "iconIMG": "",
        "lockedPanel": false,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "none",
        "dropTabCtx": "none",
        "moveRules": [],
        "moveExcludedTo": -1,
        "bookmarksFolderId": "OrEIMwiARNdN",
        "newTabBtns": [],
        "srcPanelConfig": null
      },
      "k6whdNiATW8Z": {
        "type": 1,
        "id": "k6whdNiATW8Z",
        "name": "len: 9",
        "iconSVG": "icon_bookmarks",
        "iconIMGSrc": "",
        "iconIMG": "",
        "color": "toolbar",
        "lockedPanel": false,
        "tempMode": false,
        "skipOnSwitching": false,
        "rootId": "unfiled_____",
        "viewMode": "tree",
        "autoConvert": false,
        "srcPanelConfig": null
      }
    }
  },
  "containers": [
    {
      "id": "firefox-container-1",
      "cookieStoreId": "firefox-container-1",
      "name": "8",
      "icon": "...",
      "color": "blue",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-2",
      "cookieStoreId": "firefox-container-2",
      "name": "4",
      "icon": "...",
      "color": "orange",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-3",
      "cookieStoreId": "firefox-container-3",
      "name": "7",
      "icon": "...",
      "color": "green",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-4",
      "cookieStoreId": "firefox-container-4",
      "name": "8",
      "icon": "...",
      "color": "pink",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-6",
      "cookieStoreId": "firefox-container-6",
      "name": "15",
      "icon": "...",
      "color": "blue",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    }
  ],
  "windows": [
    {
      "state": "maximized",
      "incognito": false,
      "tabsCount": 9
    }
  ],
  "bookmarks": {
    "bookmarksCount": 1970,
    "foldersCount": 107,
    "separatorsCount": 0,
    "maxDepth": 4
  }
}
```
</details>

<details>
<summary><b>Logs</b></summary>

```
No log appears.
```
</details>

**Comment by mbnuqw** [comment]
Fixed: 54518c8

**Comment by mbnuqw** [comment]
Resolved in v5.0.0
